notebooks/community/neo4j/graph_paysim.ipynb

```diff
@@ -0,0 +1,1139 @@
+{
```

Comment:
<p>AutoML Tables is no longer a standalone product in Vertex AI, so I suggest the naming "Vertex AI for AutoML tabular data."</p><p> </p><p>Reference: <a href="https://cloud.google.com/vertex-ai/docs/datasets/prepare-tabular" target="_blank">https://cloud.google.com/vertex-ai/docs/datasets/prepare-tabular</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/173/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Reply:
Done